.travis.yml

```diff
@@ -8,8 +8,21 @@ php:
   - 5.6
   - hhvm
 
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: GUZZLE_VERSION=3.*
+    - php: 5.6
+      env: GUZZLE_VERSION=4.*
+    - php: 5.6
+      env: GUZZLE_VERSION=5.*
+    - php: 5.6
+      env: GUZZLE_VERSION=6.*
```

Comment:
I think you're right but IMHO, specify each guzzle version is simpler to read.

And, with your solution, I think it's painful to manage it if php requirement change in guzzle. (e.g. Guzzle 7)